canonical/websites-content-system
# [WD-13361] show list of jira tasks on a webpage

## Done

 - Added "Summary" column to `jira_tasks` table
 - Added "Summary" optional field to the modal for requesting a new webpage
 - Rendered a list of Jira tasks associated with the given webpage

## QA

### QA steps

 - Run the app locally using dotrun
 - Open `0.0.0.0:8104` in the browser
 - Click on Request new page button in the sidebar
    - Make sure you have a VPN connection on
 - Fill in the fields that are required for "Save and generate copy doc" button to become enabled.
    - those are "URL title", "Location" and "Owner"
 - Click on the "Save and generate copy doc" button and check that the new webpage was created and appears in the tree of the sidebar (and is active)
 - Click on "Submit for publication" button, it should open the modal
 - Fill in the required fields in the modal
 - Click on "Submit" button and wait until the modal closes
 - Check that there is a Jira task rendered with the following fields listed:
    - ID
    - Summary (in case Summary field was not filled in the modal previously, this should show automatically generated text like "New webpage <name>")
    - Creation date
    - Status
 - Click on ID of the Jira task. It should open the Jira task in the new tab.

## Fixes

 - Fixes https://warthogs.atlassian.net/browse/WD-13361
